src/util/submit-addon.js

```diff
@@ -91,7 +91,7 @@ export default class Client {
     approvalCheckInterval = 1000,
     approvalCheckTimeout = 900000, // 15 minutes.
     downloadDir = process.cwd(),
-    userAgentString,
+    userAgentString = 'web-ext-lib',
```

Comment:
Do we not want to include the version number?

E.g. `web-ext-lib/<whatever is in package.json>`